api_tests/dry/rfc003_swap_reject.ts

```diff
@@ -313,55 +318,54 @@ describe("SWAP request REJECTED", () => {
 
     it("[Bob] Should be in the Rejected State after declining a swap request providing a reason", async function() {
         await bob.poll_comit_node_until(
-            chai,
             bob_stingy_swap_href,
-            body => body.state.communication.status === "REJECTED"
+            (body: SwapResponse) =>
+                body.state.communication.status === "REJECTED"
         );
     });
 
     it("[Alice] Should be in the Rejected State after Bob declines a swap request providing a reason", async () => {
         await alice.poll_comit_node_until(
-            chai,
             alice_stingy_swap_href,
-            body => body.state.communication.status === "REJECTED"
+            (body: SwapResponse) =>
+                body.state.communication.status === "REJECTED"
         );
     });
 
     it("[Bob] Can execute a decline action, without providing a reason", async () => {
-        let bob_decline_href_2;
+        let bob_decline_href_2: string;
 
         await chai
             .request(bob.comit_node_url())
             .get(bob_reasonable_swap_href)
-            .then(res => {
+            .then(async function(res) {
                 res.should.have.status(200);
                 bob_decline_href_2 = res.body._links.decline.href;
                 bob_decline_href_2.should.equal(
                     bob_reasonable_swap_href + "/decline"
                 );
-            });
+                let decline_res = await chai
```

Comment:
❌ I think it was better before not inside the `then`. Why change it?

Reply:
(oh thomas made a comment wrt to removing all then stuff which we should consider).

Reply:
because the variable may not be initialised as it is initialised inside the `then`.